dockerhub/travis-ci-image.dockerfile

```diff
@@ -14,9 +14,9 @@ RUN apt-get update -y && apt-get install -qy \
         nodejs \
         npm \
         python3 \
-        python3.7 \
+        python3.6 \
```

Comment:
why do we get an older Python here?

Reply:
don't remember exactly, but 3.7 was never supported in openstack eco-system, so testing with 3.6 and 3.8 ensures it's working there